tests/unittests/test_base_url_prefix.py

```diff
@@ -40,7 +40,7 @@ def test_no_sandbox_account_string(self, mocked_get):
         # create config
         config = {
             "access_token": "test_access_token",
-            "sandbox": "false"
+            "sandbox": False
         }
```

Comment:
If we are assuming `sandbox` type is booleon then string `true` or `false` become invalid values and hence we don't require `test_no_sandbox_account_string()` and `test_sandbox_account_string()` unit tests or need to convert those to -ve test.